ansible_base/authentication/common.py

```diff
@@ -307,4 +322,21 @@ def update_user_claims(user, database_authenticator, groups):
     if results['access_allowed'] is not True:
         logger.warning(f"User {user.username} failed an allow map and was denied access")
         return None
+
+    # We have allowed access so now we need to make the user within the system
+    reconcile_class = getattr(settings, 'ANSIBLE_BASE_AUTHENTICATOR_RECONCILE_CLASS', 'ansible_base.authentication.common')
```

Comment:
Probably `ANSIBLE_BASE_AUTHENTICATOR_RECONCILE_MODULE` instead, since it's not actually specifying the class.